src/blueprints/tv2/helpers/tv2-blueprint-configuration-mapper.ts

```diff
@@ -68,7 +68,7 @@ export class Tv2BlueprintConfigurationMapper {
       graphicsDefault: this.mapGraphicsDefault(coreConfiguration.GfxDefaults),
       graphicsSetups: this.mapGraphicsSetups(coreConfiguration.GfxSetups),
       selectedGraphicsSetup: this.findSelectedGraphicsSetup(coreConfiguration.GfxDefaults, coreConfiguration.GfxSetups),
-      dveConfigurations: this.mapDveConfigurations(coreConfiguration.DVEStyles),
+      splitScreenConfigurations: this.mapSplitScreenConfigurations(coreConfiguration.DVEStyles),
```

Comment:
What is `DVEStyles`, and i'm guessing this is not to be renamed?

Reply:
`DVEStyles` are the configuration table `DVE Layouts` in showstyle blueprints configuration. As long as we don't have control over settings, I don't think it should be renamed.